### DISCUSSION TITLE
Invalid SSL certificate

### DISCUSSION THREAD

**Discussion by PhilipDeegan** [lowlighter/metrics#1390] [Q&A]
https://metrics.lecoq.io/ 
![image](https://user-images.githubusercontent.com/30491/222850653-c5fe07bc-879a-4a92-83f9-abed02a3fe77.png)

**Comment by 0x4248** [comment]
> https://metrics.lecoq.io/ ![image](https://user-images.githubusercontent.com/30491/222850653-c5fe07bc-879a-4a92-83f9-abed02a3fe77.png)
Same here
![image](https://user-images.githubusercontent.com/60709927/222900114-83b4f83d-4f32-4cea-99a2-184f82dff5ab.png)

**Reply by 0x4248** [reply]
Created an issue at #1392

**Reply by lowlighter** [reply]
This should have been fixed now 👍 
Thanks for reporting this !

**Reply by 0x4248** [reply]
> This should have been fixed now 👍
> Thanks for reporting this !

Yeah I also sent you a discord message you can ignore that. Thank you for fixing the bug

**Reply by RMehdid** [reply]
what about the CloudFlare

**Reply by 0x4248** [reply]
What about it.

**Reply by RMehdid** [reply]
when the error occurs on the cloudFlare

**Reply by 0x4248** [reply]
![image](https://user-images.githubusercontent.com/60709927/225410752-425b01db-1f18-442f-a1c8-f22c9605f071.png)
I dont see an issue
